### PR TITLE
Add support for persistAuthorization

### PR DESCRIPTION
**Pull request by p3drosola**
SwaggerUI supports a parameter named `persistAuthorization` so that you're not logged out of the API on every reload.

https://swagger.io/docs/open-source-tools/swagger-ui/usage/configuration/#authorization

This adds optional support for that setting.

### CODE REVIEW

**Comment by MuaDDuB** [comment]
Love that one ❤️

File: README.md [hunk 1]
```diff
@@ -37,6 +37,7 @@ export default {
     parameters: {}, // OpenAPI conform parameters that are commonly used
     headers: {}, // OpenAPI conform headers that are commonly used
   },
+  persistAuthorization: true // persist authorization between reloads onthe swagger page
```
Comment (MuaDDuB) [inline README.md:40]:
```suggestion
  persistAuthorization: true // persist authorization between reloads on the swagger page
```

File: src/autoswagger.ts [hunk 2]
```diff
@@ -58,15 +60,14 @@ export class AutoSwagger {
 				<script>
 						window.onload = function() {
 							SwaggerUIBundle({
-								url: "` +
-        url +
-        `",
+								url: "` + url + `",
```
Comment (MuaDDuB) [inline src/autoswagger.ts:63]:
```suggestion
								url: "${url}",
```

Should be more readable, wdyt?

File: src/autoswagger.ts [hunk 3]
```diff
@@ -58,15 +60,14 @@ export class AutoSwagger {
...
 								dom_id: '#swagger-ui',
 								presets: [
 									SwaggerUIBundle.presets.apis,
 									SwaggerUIStandalonePreset
 								],
-								layout: "BaseLayout"
+								layout: "BaseLayout",
+                ` + persistAuthString + `
```
Comment (MuaDDuB) [inline src/autoswagger.ts:70]:
```suggestion
                ${persistAuthString}
```

Same here, should be more readable.

**Comment by ad-on-is** [comment]
@MuaDDuB I just merged your previous PRs, nice work so far, and thank you very much 👍️... If someone could fix the merge conflicts here, I can publish this one too.

**Comment by p3drosola** [comment]
thanks folks!

**Pull request merged**